### DISCUSSION TITLE
Undo migrations with down.sql?

### DISCUSSION THREAD

**Discussion by josvazg** [uptrace/bun#751] [Q&A] [answered]
Could not find in the docs whether or now using `.down.sql` scripts is supported to go back to an earlier db state.

Could you confirm whether or not that is the case?

**Comment by vmihailenco** [comment, marked as answer]
Yes, you can check [this](https://github.com/uptrace/bun/tree/master/example/migrate/migrations) example which has `.down.sql` migrations.
